com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/ConsoleProgress.java

```diff
@@ -76,6 +76,7 @@ private void printProgress() {
 
     @Override
     public void beginTask(String name, int work) {
+        System.out.print(name);
```

Comment:
Debug prints?
Won't this spam the console for _all_ resources being built in a project?
If so, it should probably be configurable, and also be done via a logger (where we can format the output)

Reply:
Isn't beginTask() only used for the main high level build steps such as "Building", "Bundling", "Cleaning" etc?

Reply:
Not sure. But if it's so, then It might probably look fine.
We really need to look into logging overall, since it's spotty at best, and also uses different logging techniques (which we cannot currently switch on or off)

Reply:
I agree. We should use Log4j or similar.